### DISCUSSION TITLE
FYI: pg-mem supports mikro-orm ... link back ?

### DISCUSSION THREAD

**Discussion by oguimbal** [mikro-orm/mikro-orm#1700] [Show and tell]
Hi !

Someone asked for mikro-orm support in pg-mem [here](https://github.com/oguimbal/pg-mem/issues/95)

[pg-mem](https://github.com/oguimbal/pg-mem) is an in-memory emulation of postgres.

It is very useful for unit testing, since it enables you to spawn blank databases and execute thousands of requests in milliseconds.
You could do this with docker or equivalent, but it's much slower to run, and to setup.

FYI, thanks to your nice extensibility model, the implementation was quite straightforward:

- I implemented [like this](https://github.com/oguimbal/pg-mem/blob/c2c3c70ec4f72356a2add3e93d56f644ed8c281f/src/adapters.ts#L300-L338) 
- You can use it [like that](https://github.com/oguimbal/pg-mem/wiki/Libraries-adapters#-mikro-orm)

Would you consider mentioning it in your documentation ?

**Comment by B4nan** [comment]
Sure, will be happy to accept PR, this should go to the recipes section.
